src/test/java/com/ratifire/devrate/service/notification/NotificationServiceTest.java

```diff
@@ -89,6 +118,75 @@ void setUp() {
         .build();
   }
 
+  private InterviewRejectedDto createInterviewRejectedDto() {
+    return InterviewRejectedDto.builder()
+        .recipientUserId(CANDIDATE_USER_ID)
+        .rejectorUserId(INTERVIEWER_USER_ID)
+        .scheduledTime(ZonedDateTime.now())
+        .recipientInterviewId(CANDIDATE_INTERVIEW_ID)
+        .rejectorInterviewId(INTERVIEWER_INTERVIEW_ID)
+        .build();
+  }
+
+  private User createInterviewerUser() {
+    return User.builder()
+        .id(INTERVIEWER_USER_ID)
+        .firstName(INTERVIEWER_FIRST_NAME)
+        .lastName(INTERVIEWER_LAST_NAME)
+        .build();
+  }
+
+  private Specialization createSpecialization() {
+    return Specialization.builder()
+        .id(1L)
+        .name(SPECIALIZATION_NAME)
+        .build();
+  }
+
+  private Skill createSkill() {
+    return Skill.builder()
+        .id(1L)
+        .name(SKILL_NAME)
+        .build();
+  }
+
+  private Mastery createCandidateMastery() {
+    return Mastery.builder()
+        .id(CANDIDATE_MASTERY_ID)
+        .specialization(createSpecialization())
+        .skills(List.of(createSkill()))
+        .build();
+  }
+
+  private Mastery createInterviewerMastery() {
+    return Mastery.builder()
+        .id(INTERVIEWER_MASTERY_ID)
+        .specialization(createSpecialization())
+        .softSkillMark(BigDecimal.valueOf(4.5))
+        .hardSkillMark(BigDecimal.valueOf(4.8))
+        .build();
+  }
+
+  private Interview createCandidateInterview(ZonedDateTime interviewDateTime) {
+    return Interview.builder()
+        .id(CANDIDATE_INTERVIEW_ID)
+        .userId(CANDIDATE_USER_ID)
+        .masteryId(CANDIDATE_MASTERY_ID)
+        .role(InterviewRequestRole.CANDIDATE)
+        .startTime(interviewDateTime)
+        .build();
+  }
+
+  private Interview createInterviewerInterview(ZonedDateTime interviewDateTime) {
+    return Interview.builder()
+        .id(INTERVIEWER_INTERVIEW_ID)
+        .userId(INTERVIEWER_USER_ID)
+        .masteryId(INTERVIEWER_MASTERY_ID)
+        .role(InterviewRequestRole.INTERVIEWER)
+        .startTime(interviewDateTime)
+        .build();
```

Comment:
all this method need to be removed. If you want to reuse objects consider to create them in beforTest or beforeClass.